.gitignore

```diff
@@ -27,6 +27,7 @@ scons-local*
 .sconsign.dblite
 /*.d
 /vcpkg/
+tbb/
```

Comment:
can this please go into a subfolder named external or something alike, I know this is a temporary thing, but I fear that other libs might do the same thing and it would get really crowded in my es folder?

Reply:
Sure, I can rename it. Btw, if you have access to a windows machine, can you check if you can get it to build by extacting tbb to the same place where the mingw libraries are (and tweaking cmake config to account for it)? That would mean only the CI would have to use this workaround.

~~Well it's not any worse than having catch's source code in our tree~~

Reply:
I do not have a windows machine, sorry